db/data/regions/AT/en.yml

```diff
@@ -5,13 +5,15 @@ en:
       addresses:
         address1:
           label:
-            default: Street and house number
-            optional: Street and house number (optional)
+            default: Province
+            optional: Province (optional)
           errors:
-            blank: Enter a street and house number
+            blank_instructional: Enter a province
+            blank_informative: Province is missing
         address2:
           label:
             default: Additional address
             optional: Additional address (optional)
           errors:
-            blank: Enter additional address information
+            blank_instructional: Enter additional address information
+            blank_informative: Additional address information is missing
```

Comment:
Do we know what's expected here? If a buyer sees this, they might understand what to do, but it doesn't feel very "informative" if this is optional and open-ended in the first place.